tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/SeleniumTests.cs

```diff
@@ -0,0 +1,163 @@
+// <copyright file="SeleniumTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+using Datadog.Trace.Ci.CiEnvironment;
+using Datadog.Trace.Ci.Tags;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.ExtensionMethods;
+using Datadog.Trace.TestHelpers;
+using Datadog.Trace.TestHelpers.Ci;
+using Datadog.Trace.Util;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
+using FluentAssertions;
+using VerifyXunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.CI;
+
+[UsesVerify]
+public class SeleniumTests : TestingFrameworkEvpTest
+{
+    public SeleniumTests(ITestOutputHelper output)
+        : base("Selenium", output)
+    {
+        SetServiceName("xunit-selenium-tests");
+        SetServiceVersion("1.0.0");
+    }
+
+    [SkippableTheory]
+    [MemberData(nameof(PackageVersions.Selenium), MemberType = typeof(PackageVersions))]
+    [Trait("RunOnWindows", "True")]
+    [Trait("Category", "EndToEnd")]
+    [Trait("Category", "TestIntegrations")]
+    public async Task Injection(string packageVersion)
+    {
+        SkipOn.Platform(SkipOn.PlatformValue.Linux);
+        SkipOn.Platform(SkipOn.PlatformValue.MacOs);
+        var tests = new List<MockCIVisibilityTest>();
+        var testSuites = new List<MockCIVisibilityTestSuite>();
+        var testModules = new List<MockCIVisibilityTestModule>();
+
+        // Inject session
+        var sessionId = RandomIdGenerator.Shared.NextSpanId();
+        var sessionCommand = "test command";
+        var sessionWorkingDirectory = "C:\\selenium\\working_directory";
```

Comment:
Do we have to use `C:\\selenium`? This seems like it could cause some pain, particularly for CI... Can we use something inside the repository, like `/tracer/build_data` instead, or the temp directory or something?

Reply:
this only a dummy value, I can put whatever (valid path format)

Reply:
Changed to `Path.GetTempPath()` in https://github.com/DataDog/dd-go/pull/132491/files